docs/modules/globalfee/README.md

```diff
@@ -7,9 +7,10 @@ The CosmosHub has two types of fees, both of which, are the [sdk.DecCoins](https
 - min_gas_prices (are specified validator node, in the `config/app.toml` configuration file)
   
 ### Global Fees
-Global fees are set up through governance proposal which must be voted on by validators. 
+#### Global Fees Concept
+Global fees are the fees that each transaction (except [bypass fee message types](###Bypass Fees Message Types)) has to satisfy. Global fees are set up through governance proposal which must be voted on by validators. 
```

Comment:
Perhaps change:
_Global fees are the fees that each transaction has to satisfy_ **to** Global fees are the fees that each transaction incurs

Reply:
```suggestion
Global fees are the fees that each transaction incurs—except [bypass fee message types](###Bypass Fees Message Types). Global fees are set up through a governance proposal which must be voted on by validators. 
```